### ISSUE TITLE
Icon for SettingToggle

### ISSUE THREAD

**Issue by project-academy** [aheze/Setting#24]
Is it possible to have a icon on SettingToggle in the same way that SettingPicker and SettingPage do?
I see you've created #23 that will allow for lots more composability of these views, but I just wanted to double check this wasn't already possible. 

Thanks so much for making this btw, I know you're a young guy, and you've got tremendous amounts of talent! You've helped me have better Settings options more my users, and helped me to start implementing SwiftUI in my UIKit app 😊

**Comment by aheze** [comment]
Yeah, currently not supported - should be an easy addition though. I'll get back to this when I get the time

**Comment by project-academy** [comment]
Hi, do you know when you might get back to this?
